convert-stage-4.sh

```diff
@@ -154,8 +155,11 @@ install_files() {
   fi
 
   # Enable menderd service starting on boot.
-  sudo ln -sf /lib/systemd/system/mender.service \
-      ${primary_dir}/etc/systemd/system/multi-user.target.wants/mender.service
+  if ! [ -z "${no_service}" ]; then
```

Comment:
Is this correct? I assume you want to enable the service only if no_service not set (i.e. `test -z`).

Reply:
You're completely right